### DISCUSSION TITLE
Can we choose our preferred format to display the date and time?

### DISCUSSION THREAD

**Discussion by Mishasama** [community/community#62361] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm really not used to this American-style date display, it's so unintuitive to me that I have to reverse the order every time to think about what day it actually says.

**Comment by simonmcconnell** [comment]
I was checking a branch and it said the last commit was in `Feb 23`, which means the month February in the year 2023 to me.  What you're trying to say to me, an Australian, is 23 Feb.
